htmltools/_core.py

```diff
@@ -1270,6 +1262,31 @@ def as_string(self) -> str:
         return self + ""
 
 
+class HTML(RAW):
+    """
+    Mark a string as raw HTML. This will prevent the string from being escaped when
+    rendered inside an HTML tag.
+
+    Examples
+    --------
+    >>> from htmltools import HTML, div
+    >>> div("<p>Hello</p>")
+    <div>&lt;p&gt;Hello&lt;/p&gt;</div>
+    >>> div(HTML("<p>Hello</p>"))
+    <div><p>Hello</p></div>
+    """
+
+    pass
+
+
+class JS(RAW):
+    """
+    Mark a string as raw JavaScript.
```

Comment:
```suggestion
    Mark a string as raw JavaScript. This will prevent the string from being escaped when
    rendered inside an HTML tag.
```

Reply:
Wait, is this not true? Now that I think about it, maybe it shouldn't be true? Or at least, `JS()` for the purposes of using inside `tags.script()` is very different than `JS()` for the purposes of marking certain parts of JSON-encoded options as needing to be evaluated in the client.